src/lib/ngsi/StringList.cpp

```diff
@@ -65,20 +65,20 @@ void AttributeList::fill(const std::string& commaSeparatedList)
 *
 * render - 
 */
-std::string AttributeList::render(bool comma)
+std::string StringList::render(bool comma)
 {
   std::string  out = "";
 
-  if (attributeV.size() == 0)
+  if (stringV.size() == 0)
   {
     return "";
   }
 
-  out += startTag("attributes", true);
+  out += startTag("strings", true);
```

Comment:
Not sure about this change... Whoi is using this render() method? The NGSIv1 responses rendering? Has you tested full functional test regression after the changes?

Reply:
I had not tried all the tests yet. I've modified the render method, accordingly to what 'ive done in parseStringList() method, that is to add the fieldName parameter to the render() method.
Now everything seems ok and tests passed, fixing with be6a06c